src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -315,12 +317,16 @@ protected virtual void Initialize()
                 List<FunctionDescriptorProvider> descriptionProviders = new List<FunctionDescriptorProvider>()
                 {
                     new ScriptFunctionDescriptorProvider(this, ScriptConfig),
+#if FEATURE_NODE
```

Comment:
If you compress all these flags to one (see comments below), then we can simplify this code to just a single `#if !MONO` block